swap_meet/item.py

```diff
@@ -1,2 +1,17 @@
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, category = "", condition = 0):
+        self.category = category
+        self.condition = condition
+
+    def __str__(self):
+        return "Hello World!"
+
+    def condition_description(self):
+        if self.condition == 5:
```

Comment:
This works perfectly well, but I thought I'd mention an alternative strategy! Explicitly writing this conditional logic does give us a lot of control over exactly under which situations a particular description should be used, but especially when we have a long list of chained checks that are very similar, we could try coming up with a way to represent it with data and access it by index. A basic example of this that would mirror your existing behavior could look like:
```
# outside class definition
DESCRIPTIONS = (
    "Are you sure you want this?",
    "Probably best to clean this one first....",
    "Pretty minty, if you ask me",
    "Are you sure this is second-hand?"
)

# in Item class definition
    def condition_description(self):
        return DESCRIPTIONS[self.condition]
```
That doesn't capture your ranges, but you could use `floor` or `ceiling` or 'round`. And we would still want to think about the edge case behaviors (outside the expected range? non-integer conditions, etc.), but we would have more general confidence that if this logic works for a particular condition, it's likely to work for all of them.